slurm_ops_manager/slurm_ops.py

```diff
@@ -68,6 +68,46 @@ def slurm_component(self) -> str:
         """Return the slurm component."""
         return self._slurm_resource_manager.slurm_component
 
+    @property
+    def fluentbit_config_nhc(self) -> list:
+        """Return Fluentbit configuration parameters to forward NHC logs."""
+        cfg = [{"input": [("name",             "tail"),
+                          ("path",             "/var/log/nhc.log"),
+                          ("path_key",         "filename"),
+                          ("tag",              "nhc"),
+                          ("multiline.parser", "nhc")]},
+               {"multiline_parser": [("name",          "nhc"),
+                                     ("type",          "regex"),
+                                     ("flush_timeout", "1000"),
+                                     ("rule",          '"start_state"', '"/^([\d]{8} [\d:]*) (.*)/"', '"cont"'), # noqa
+                                     ("rule",          '"cont"',        '"/^([^\d].*)/"',             '"cont"')]}] # noqa
+        return cfg
+
+    @property
+    def fluentbit_config_slurm(self) -> list:
+        """Return Fluentbit configuration parameters to forward Slurm logs."""
+        if self._slurm_component == "slurmd":
```

Comment:
I wonder if it would be worth it to collapse this down and get the attribute programatically:
```python
if self._slurm_component == "slurmrestd":
    return
log_file = getattr(self._slurm_resource_manager, f"_{self._slurm_component}_log_file")
```

Reply:
I never really understood how `getattr` works... What about this instead:
```python
log_file = eval(f"self._slurm_resource_manager.{self._slurm_component}_log_file")
```

On the other hand: the zen of Python says "explicit is better than implicit"...

Reply:
`eval` can be dangerous, I like the way it is now, if we keep adding more things here, I'd go with Tucker's suggestion.